fixed: Add command to create build directory in Makefile

sigmadream opened the pull request:
- Bug that recognizes the build directory as a file

```
$ (venv) ➜  snakebarrel git:(fixed/Makefile) ✗ make linux
	...
PythonAction Error
Traceback (most recent call last):
  File "/home/sd/works/snakebarrel/venv/lib/python3.10/site-packages/doit/action.py", line 461, in execute
    returned_value = self.py_callable(*self.args, **kwargs)
  File "/home/sd/works/snakebarrel/venv/lib/python3.10/site-packages/doit/tools.py", line 20, in create_folder
    os.makedirs(dir_path, exist_ok=True)
  File "/usr/lib/python3.10/os.py", line 225, in makedirs
    mkdir(name, mode)
NotADirectoryError: [Errno 20] Not a directory: '/home/sd/works/snakebarrel/build/_output'

make[1]: *** [Makefile:26: compile] Error 2
make[1]: Leaving directory '/home/sd/works/snakebarrel'
make: *** [Makefile:30: dep] Error 2
```

- After creating the build directory, add a directory creation command to proceed with make

```
$ (venv) ➜  snakebarrel git:(fixed/Makefile) ✗ make linux
	...

[BUILD] Done.

cd app; zip -r -9 ./SnakeBarrel-linux-x64-221004.zip "./SnakeBarrel-linux-x64"

[BUILD] Done.

	...

cd app; zip -r -9 ./SnakeBarrel-linux-arm64-221004.zip "./SnakeBarrel-linux-arm64"

mv ./app/SnakeBarrel-linux-arm64-221004.zip ./app/SnakeBarrel-0.1.0-linux-arm64.zip
```

* System: Linux sd-amd3600 5.15.0-48-generic #54-Ubuntu SMP Fri Aug 26 13:26:29 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux

Thx!

sigmadream committed:
fixed: Add command to create build directory in Makefile

- Bug that recognizes the build directory as a file
- After creating the build directory, add a directory creation command to proceed with make

sigmadream commented:
@inureyes May I ask for a code review?

inureyes commented on Makefile:
```diff
@@ -22,6 +22,7 @@ compile: versiontag
 # xeus-based kernel
 #	python -m jupyter lite build --XeusPythonEnv.packages=numpy,matplotlib,ipyleaflet,scipy,pandas,scikit-learn,scikit-image,networkx --lite-dir=build
 #	cp ./overrides.json ./build
+	mkdir build
```
`mkdir -p build` would be better since `-p` option ignores if directory exists.

sigmadream replied:
I found that '-p' is used to create hierarchical directories. I'm glad to know this new 'ability'! Thx!

inureyes replied:
LGTM

inureyes reviewed (changes requested):
`mkdir -p build` would be better since `-p` option ignores if directory exists.

sigmadream committed:
fixed: '-p' option of the mkdir command.

- An error does not occur when there is a directory created by using the  '-p' option of the mkdir command.
